.github/workflows/push-mimir-build-image.yml

```diff
@@ -0,0 +1,108 @@
+name: Build and Push mimir-build-image
+
+# configure trigger by pull request or manually
```

Comment:
I don't think this workflow can be triggered manually - don't we need a `workflow_dispatch` trigger?